resources/views/guest/mainPage.blade.php

```diff
@@ -0,0 +1,255 @@
+                                </div>
+                            </div>
+                        @elseif($index < 15)
+                            <div class="col-lg-4  collapse show" id="category-jobs">
+                                <h5 class=" text-bottom p-2"><a href="#">{{$industry->name}}Jobs</a></h5>
+                            </div>
+                        @endif
+                    @endforeach
+                    <div class="col-sm-12">
+                    <button class=" btn btn-primary  w-100 d-lg-none justify-content-center d-flex" type="button" data-toggle="collapse" data-target="#category-jobs" aria-controls="category-jobs" aria-expanded="true" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                        show more
+                    </button>
+                    </div>
+                </div>
+            @else
+                <div class="container text-center">
+                    <h1>There is no Industry Category Yet</h1>
+                </div>
+            @endif
+        </div>
+        <hr class="my-5 mx-5">
+        {{--  Browse Jobs by Location --}}
+        <div class="jobs-by-category my-5">
+            <h1 class="mt-5 mb-3 font-weight-bold text-secondary">Browse Jobs by Location</h1>
+           @if ($cities->count() > 0)
+                <div class="row ">
+                    @foreach ($cities as $index => $cit)
+                        @if($index < 3)
+                            <div class="col-lg-4 col-sm-12 my-4">
+                                <div class="box-shad">
+                                    <a href="#">
+                                        <img class="career-level" src="{{asset('img/guest/city/'.$city[$index])}}" alt="Not Found">
+                                    </a>
+                                    <h5 class="position-absolute text-light text-bottom p-2">Jobs in {{$cit->name}}</h5>
+                                </div>
+                            </div>
+                        @elseif($index < 15)
+                        <div class="col-lg-4 collapse show" id="city-jobs">
+                            <h5 class=" text-bottom p-2"><a href="#">Jobs in {{$cit->name}}</a></h5>
+                        </div>
+                        @endif
+                    @endforeach
+                <div class="col-sm-12">
+                    <button class=" btn btn-primary  w-100 d-lg-none justify-content-center d-flex" type="button" data-toggle="collapse" data-target="#city-jobs" aria-controls="city-jobs" aria-expanded="true" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                        show more
+                    </button>
+                </div>
+                </div>
+            @else
+                <div class="container text-center">
+                    <h1>There is no cities Yet</h1>
+                </div>
```

Comment:
If this is possible and there are no cities.
Don't show this section only, don't tell the user.